### ISSUE TITLE
support builder utils (and export them) for gsplot users?

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#220]
This was brought up by @thongsav-usgs and maybe a bit by @jiwalker-usgs, but we might want to make simpler ways to build the gsplot objects when the use-cases are more complex than our basic ones that are in the docs. 

PT, want to weigh in here?

**Comment by jiwalker-usgs** [comment]
Is this similar to #193 but including exposing some of the logic for building this?

**Comment by jordansread** [comment]
no, I was actually thinking it would be a different thing. 

Basically, we use `append` internally, but don't expose (for example) the `after` arg to `append`. This could support ordering of the gsplot object. 

Also, the thing PT was doing with do.call could perhaps be better supported by exposing some of the internal name matching. Just a thought.

**Comment by lindsayplatt** [comment]
@jread-usgs you implemented a method to order the gsplot object via #378. Is there more we should do for this issue?

**Comment by jordansread** [comment]
I don't think so. Does it meet your expectation for what the user(s) need access to @lindsaycarr ?

**Comment by lindsayplatt** [comment]
Since only the ordering of the gsplot object was specifically called out here, I'd say yes, it works. The way we use do.call in repgen seems very specific, not sure others would benefit from it.
